api/hearings/models/hearingStatusMappings.ts

```diff
@@ -35,7 +35,7 @@ export const hearingStatusMappings: HearingStatusMapping[] = [
   },
   {
     hmcStatus: HMCStatus.CANCELLATION_REQUESTED,
-    exuiSectionStatus: EXUISectionStatusEnum.UPCOMING,
+    exuiSectionStatus: EXUISectionStatusEnum.PAST_AND_CANCELLED,
```

Comment:
It should still be EXUISectionStatusEnum.UPCOMING if it's HMCStatus CANCELLATION_REQUESTED, could you double check the Status table?

Reply:
https://tools.hmcts.net/confluence/display/CHMC/Hearing+Status

Reply:
The ticket expected this result

Reply:
'And I see the 'Cancelled Hearing' in the past and cancelled hearing section' 

This means after the API is called, the hearing list is refreshed and changed, but it's nothing related to the mapping table, this mapping table needs to be exactly consistent with: https://tools.hmcts.net/confluence/display/CHMC/Hearing+Status.

As we are using mock, so there is a limitation that we can't show the heaing is changed to 'Cancelled Hearing' and moved to 'past and cancelled', but we can demonstrate that the page is refreshed and a new mock API is called to refresh the hearings list tab, though the mock API is still returning the same hearing list data but that is fine.

Let's discuss tmr.